docs/GuiTestDocumentation.md

```diff
@@ -0,0 +1,282 @@
+2. Event should be added and should be ordered by date
+
+|                           Before                           |                          After                           |
+|:----------------------------------------------------------:|:--------------------------------------------------------:|
+| ![BeforeAddEvent](images/GUITestImages/BeforeAddEvent.png) | ![AfterAddEvent](images/GUITestImages/AfterAddEvent.png) |
+
+As you can see in the above images `WL's Bday` was inserted in the correct order with the appropriate command result message displayed in the CommandResult box.
+
+If incorrect format of `addevent` command is entered, the appropriate error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![AddEventError](images/GUITestImages/AddEventError.png)
+
+#### Edit an event
+
+Upon successful editing of an event
+
+1. Correct Command result display message
+2. Event should be edited and if date is changed, list should be reordered
+
+|                            Before                            |                           After                            |
+|:------------------------------------------------------------:|:----------------------------------------------------------:|
+| ![BeforeEditEvent](images/GUITestImages/BeforeEditEvent.png) | ![AfterEditEvent](images/GUITestImages/AfterEditEvent.png) |
+
+As you can see in the above images `WL's Bday` date was edited to a later date, thus the list was reordered. The appropriate command result message displayed was also in the CommandResult box.
+
+If incorrect format of edit command is entered, the appropriate edit friend error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![EditEventError](images/GUITestImages/EditEventError.png)
+
+#### Deleting an event
+
+Upon successful deletion of an event
+
+1. Correct Command result display message
+2. Event card should be removed from the GUI
+
+|                              Before                              |                               After                                |
+|:----------------------------------------------------------------:|:------------------------------------------------------------------:|
+| ![BeforeDeleteEvent](images/GUITestImages/BeforeDeleteEvent.png) | ![AfterDeleteEvent.png](images/GUITestImages/AfterDeleteEvent.png) |
+
+As you can see in the above images `Dione's Bday` was deleted with the appropriate command result message displayed in the CommandResult box.
+
+If incorrect format of `deleteevent` command is entered, the appropriate delete event error message should be displayed in the CommandResult box. Shown below is the error message in the CommandResultBox.
+
+![DeleteEventError](images/GUITestImages/DeleteEventError.png)
+
+#### Show all events
+
+Upon successful execution of `showfriends` command the following must occur
```

Comment:
I think you mean `events` here instead of friends!!

Reply:
Changed!